resources/js/ApiPageConnectionRepo.js

```diff
@@ -10,9 +10,10 @@ module.ApiPageConnectionRepo = ( function ( mw, ApiConnectionsBuilder ) {
 
 	/**
 	 * @param {string[]} pageNames
+	 * @param {boolean} enableDisplayTitle
 	 * @return {Promise}
 	 */
-	ApiPageConnectionRepo.prototype.addConnections = function(pageNames) {
+	ApiPageConnectionRepo.prototype.addConnections = function(pageNames, enableDisplayTitle) {
```

Comment:
I have the impression this should be a constructor parameter instead. I don't recall the wiring of the extension though so cannot say for sure that is possible without checking.

Reply:
You can indeed inject it in the constructor rather than passing it to `Network` and all those functions here.

Reply:
Done.